projects/packages/my-jetpack/_inc/data/types.ts

```diff
@@ -28,3 +28,11 @@ export type ProductCamelCase = ToCamelCase< ProductSnakeCase > & {
 		discountPricePerMonth: number;
 	};
 };
+
+export type WP_Error = {
+	code: string;
```

Comment:
If we are not sure / we don't care about other codes, we can at least declare here in the type ones we are using in the code so we will get a type help when developing.

```suggestion
	code: 'not_connected' | string;
```

Reply:
Good idea, thanks!